specification/gedcom-3-structures-1-organization.md

```diff
@@ -169,6 +171,20 @@ A few substructures of note:
     - `HEAD`.`SOUR`.`DATA` describes a larger database this data is extracted from.
 - `LANG` and `PLAC` give a default value for the rest of the document.
 
+### New for 7.1: 
+- `PLAC` now is {1:1} so obligatory.
+- `FORM` now has `g7:HEAD-PLAC-FORM71` which is defined as follows:  
+The `<List:Text> ` always consists of the following string of jurisdictions (smallest to largest:  
+  **`LOCATION, ZIPCODE, VILLAGE, CITY, CODEINSEE, DISTRICT, PROVINCE, COUNTY, STATE, COUNTRY, SEA, EARTH`**  
+  These are in fact the original jurisdictions from GEDCOM7 with a few added. With this list, older GEDCOM files can be converted, by comparing their `PLAC.FORM's` with this list.  
```

Comment:
There are no "original jurisdictions in GEDCOM 7" per se.  There are two examples, but those are merely examples, not normative.

Reply:
and this becomes:

These are in fact jurisdictions found in current GEDCOM files, with a few added. With this list, older GEDCOM files can be converted, by comparing their `PLAC.FORM's` with this list.